samples/custom-recoverability/Core_8/Server/Program.cs

```diff
@@ -0,0 +1,59 @@
+using NServiceBus;
+using NServiceBus.Transport;
+using System;
+using System.Threading.Tasks;
+
+class Program
+{
+    static async Task Main()
+    {
+        Console.Title = "Samples.CustomRecoverability.Server";
+        var endpointConfiguration = new EndpointConfiguration("Samples.CustomRecoverability.Server");
+        endpointConfiguration.UsePersistence<LearningPersistence>();
+        endpointConfiguration.UseTransport(new LearningTransport());
+        var recoverability = endpointConfiguration.Recoverability();
+
+        #region disable
+        //recoverability.Delayed(settings =>
+        //{
+        //    settings.NumberOfRetries(0);
+        //});
```

Comment:
Why is this commented out?

Reply:
These are commented out to begin with and the sample doc has instructions when to uncomment them and what happens when its done.